doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc

```diff
@@ -60,7 +56,7 @@ ifdef::PAM[]
 ** `h2`: A built-in `h2` database engine that does not require a separate pod. Do not scale the {KIE_SERVER} pod if you use this setting.
 ** `external`: An external database server. 
 . If you selected any database except `external`, a Persistent Volume Claim will be created to store the database. Optionally, in the *Size* field, enter the size of the persistence volume to create.
-. If you selected the `external` database and you want to use any database server except PostgreSQL or MySQL, you must provide a {KIE_SERVER} extension image with the database server driver according to instructions in <<externaldb-build-proc>>. To configure the {KIE_SERVER} to use this extension image, make the following changes:
+. If you selected the `external` database, optionally configure the {KIE_SERVER} extension image. If you want to use any database server except PostgreSQL, MySQL, or MariaDB, you must provide a {KIE_SERVER} extension image with the database server driver according to instructions in <<externaldb-build-proc>>. To configure the {KIE_SERVER} to use this extension image, make the following changes:
```

Comment:
Minor suggestion - Move optionally and rephrase:

Optionally, if you selected the `external` database, configure the {KIE_SERVER} extension image.